src/modals/AddressDetailsModal.tsx

```diff
@@ -100,44 +96,20 @@ const AddressDetailsModal = ({ addressHash, onClose }: AddressDetailsModalProps)
         </AmountsOverviewPanel>
         <Shortcuts>
           <ButtonsGrid>
-            <ShortcutButton
-              transparent
-              borderless
-              onClick={() => setIsReceiveModalOpen(true)}
-              Icon={ArrowDown}
-              iconColor={theme.global.valid}
-            >
-              <ButtonText>{t('Receive')}</ButtonText>
-            </ShortcutButton>
-            <ShortcutButton
-              transparent
-              borderless
-              onClick={() => setIsSendModalOpen(true)}
-              Icon={ArrowUp}
-              iconColor={theme.global.accent}
-            >
-              <ButtonText>{t('Send')}</ButtonText>
-            </ShortcutButton>
-            <ShortcutButton transparent borderless onClick={() => setIsAddressOptionsModalOpen(true)} Icon={Settings}>
-              <ButtonText>{t('Settings')}</ButtonText>
-            </ShortcutButton>
+            <ShortcutButtons receive send addressSettings addressHash={address.hash} />
```

Comment:
The same buttons were repeated in 3 different places, so I extracted them all to one component.